libs/perun/components/src/lib/perun-header/perun-header.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import {
+  NotificationStorageService,
+  OtherApplicationsService,
+  StoreService,
+} from '@perun-web-apps/perun/services';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { TranslateService } from '@ngx-translate/core';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { MatDialog } from '@angular/material/dialog';
+import { ShowNotificationHistoryDialogComponent } from '@perun-web-apps/perun/dialogs';
+
+export enum AppType {
+  Admin = 'admin',
+  Profile = 'profile',
+  PwdReset = 'pwdReset',
+  Consolidator = 'consolidator',
+  Linker = 'linker',
+}
```

Comment:
It would be also fine if you created this enum/type in `libs/perun/models/src/lib/` and reuse it here and also in the otherApplicationService. Now the method `getUrlForOtherApplication()` gets as a parameter `appType`, but the type of this parameter is defined again - there can be used the same type imported from `models`.